src/example/example.service.ts

```diff
@@ -0,0 +1,93 @@
+export class ExampleService {
+  public calculateAverage(numbers: number[]): number {
+    if (!numbers.length) return 0;
+    let sum = 0;
+    for (const num of numbers) {
+      sum += num;
+    }
+    return sum / numbers.length;
```

Comment:
PullProofAI

Consider using the `reduce` method for calculating the sum of numbers in `calculateAverage` to simplify the logic and improve readability.
```diff
+ let sum = numbers.reduce((acc, num) => acc + num, 0);
```



<!-- This is an auto-generated comment by PullProofAI -->